superset/security.py

```diff
@@ -99,11 +117,18 @@ def sync_role_definitions():
 
     logging.info("Syncing admin perms")
     for p in perms:
-        sm.add_permission_role(admin, p)
+        # admin has all_database_access and all_datasource_access
+        if is_user_defined_permission(p):
```

Comment:
are you trying to delete all the perms knowing they get recreated if missing later on?